docker/docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+data:
+    image: busybox
+    volumes:
+        - ../:/var/www/app
+
+nginx:
+    image: yappabe/nginx
```

Comment:
:+1: for official nginx and php images. 

I also recommend to use `php:5.6-apache`. Like this you have only one container for the `web`. I think it's easier to maintain.

What do you think ?